docker-compose.kafka-queue.yml

```diff
@@ -0,0 +1,110 @@
+version: "3.2"
+services:
+    gateway:
+        volumes:
+            - "/var/run/docker.sock:/var/run/docker.sock"
+        ports:
+            - 8080:8080
+        image: functions/gateway:latest-dev
+        networks:
+            - functions
+        environment:
+            dnsrr: "true"  # Temporarily use dnsrr in place of VIP while issue persists on PWD
+            faas_kafka_brokers: kafka1:19092,kafka2:19092
+            faas_queue_topics: faas-request # ,other-topics
+        depends_on:
+            - kafka1
+            - kafka2
+        deploy:
+            restart_policy:
+                condition: on-failure
+                delay: 5s
+                max_attempts: 20
+                window: 380s
+            placement:
+                constraints:
+                    - 'node.role == manager'
+                    - 'node.platform.os == linux'
+
+
+    kafka-queue-worker:
+        image: functions/kafka-queue-worker:latest-dev
+        networks:
+            - functions
+        environment:
+            faas_kafka_brokers: kafka1:19092,kafka2:19092
+            faas_queue_topics: faas-request # ,other-topics
+        depends_on:
+            - kafka1
+            - kafka2
+        deploy:
+            restart_policy:
+                condition: on-failure
+                delay: 5s
+                max_attempts: 20
+                window: 380s
+            placement:
+                constraints:
+                    - 'node.platform.os == linux'
+
+    # End
+
+    prometheus:
```

Comment:
I wonder if we run one file after another, whether that can avoid having to repeat all the other services that are the same in both files?